src/Core/View.js

```diff
@@ -278,4 +279,49 @@ View.prototype.getLayers = function getLayers(filter) {
     return result;
 };
 
+/**
+ * @typedef {object} FrameRequester
+ * @property {Function(dt, firtLoop)} update - Method that will be called each
```

Comment:
typo: `firtLoop` => `firstLoop`. Maybe `updateLoopRestarted` is a better name too?

I'm still wondering if this boolean is really useful..